packages/esm-patient-orders-app/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const moduleName = '@openmrs/esm-patient-orders-app';
+export const medicationsOrderBasket = 'medications';
+export const labsOrderBasket = 'labs';
```

Comment:
This was intentional because i foresaw some needs with some other actions and hadn't scoped out the need for other actions. Might remove these in follow up PR's if it's not shared